pixi.toml

```diff
@@ -0,0 +1,34 @@
+# This file was generated automatically from conda-smithy. To update this configuration,
+# update the conda-forge.yml and/or the recipe/meta.yaml.
+# -*- mode: toml -*-
+
+[project]
+name = "uproot-feedstock"
+version = "3.46.1"
```

Comment:
Just curious, what is this number?

Reply:
Must be some version from the generation step.

Reply:
About to go AFK, so I also don't remember off the top of my head how that gets set. But I can look it up.

Reply:
@henryiii Ah I am pretty sure it is the `conda-smithy` version used. c.f. https://github.com/conda-forge/conda-smithy/releases/tag/v3.46.1